README.md

```diff
@@ -1,10 +1,12 @@
 # generator-aspnet
 
+[![Join the chat at https://gitter.im/jackjwilliams/generator-aspnet-semanticui](https://badges.gitter.im/jackjwilliams/generator-aspnet-semanticui.svg)](https://gitter.im/jackjwilliams/generator-aspnet-semanticui?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
This is not badge for this repo
:-1: 
The OmniSharp has just its own `Omnisharp` general repo for issues/talks.